apps/src/javalab/Backpack.jsx

```diff
@@ -212,6 +212,12 @@ class Backpack extends Component {
       !backpackFilesLoading &&
       !backpackLoadError &&
       backpackFilenames.length === 0;
+    const hasSelectedFiles = selectedFiles.length !== 0;
+
+    const importButtonStyles = {...styles.importButton};
+    if (hasSelectedFiles) {
+      importButtonStyles.borderColor = color.orange;
```

Comment:
would it make more sense for this to be a `:disabled` style on `styles.importButton`?

Reply:
didn't realize that was a thing, can do that!

Reply:
`PaneButton` doesn't use the `disabled` attribute directly so I feel like it might be better to leave as-is? Does a `disabled` selector match elements that don't have an onclick?

https://github.com/code-dot-org/code-dot-org/blob/2670dc432709ae0b7c842f7d80d1624a74e7bc35/apps/src/templates/PaneHeader.jsx#L195-L205

Reply:
Ah maybe `:disabled` is broader than I thought, just picks up anything you can't interact with:

https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled